.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java-version: [8, 11, 12, 13]
```

Comment:
Why remove java 8 support? It's the version of the most users use. It's a really bad idea to remove it. Plus, it doesn't make sense to test 4 versions (8, 11, 12, 13), only 8, 11, and 13 should be test.

If you really want to use ``Optional#isEmpty`` method (introduced in java 11), you should use the negation of ``Optional#isPresent`` instead.

Reply:
I removed Java 8 support because java 9-11 have very useful features, and from what I know, updating to Java 11 isn't that big of an endeavor (I have been told it's very well possible on minecraft servers for example).

Regardless, if this project is truly meant to have scope beyond Minecraft, what should be used is the latest stable (i.e long-term support) version, which is Java 11.

Optional#isEmpty is not the only new method I'm using, Optional#or, String#repeat and String#strip are other methods that make the code cleaner and better.

At the end of the day, we could afford staying on Java 8 ; it's only a couple methods less with rather easy replacements. But the actual question is : should we care?

And as for the 4 different versions, don't ask me, I didn't make these.